# This msg file specifies the dimension (width(x), length(y), height(z)) of a 3D bounding box
# e.g., a mesh of a detected object

float32 x
float32 y
float32 z
